3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_mop_up.psql

```diff
@@ -184,12 +211,16 @@ SELECT * FROM
             WHEN coverage_query.sprayCov = 0 THEN 'n/a'
             WHEN coverage_query.sprayCov > 0.9 THEN '0'
             ELSE 'Not done'
-        END AS reviewed_with_decision
-    FROM plans
+        END AS reviewed_with_decision,
+        irs_sa_events.latest_event_id AS latest_sa_event_id,
+        irs_sa_events.latest_event_date AS latest_sa_event_date
+    FROM pending.restricted_plans AS plans
     LEFT JOIN zambia_plan_jurisdictions
         ON plans.identifier = zambia_plan_jurisdictions.plan_id
     LEFT JOIN zambia_jurisdictions
         ON zambia_plan_jurisdictions.zambia_jurisdiction_id = zambia_jurisdictions.jurisdiction_id
+    LEFT JOIN jurisdictions AS health_center_jurisdictions
+    	ON health_center_jurisdictions.id = zambia_jurisdictions.jurisdiction_path[4]
```

Comment:
This is about as good as we can do without sync'ing down locationTag metadata from jurisdictions.